src/backend/marsha/core/serializers.py

```diff
@@ -299,6 +302,32 @@ class Meta:  # noqa
     thumbnail = ThumbnailSerializer(read_only=True, allow_null=True)
     urls = serializers.SerializerMethodField()
     is_ready_to_show = serializers.BooleanField(read_only=True)
+    disabled_use_subtitle_as_transcript = serializers.SerializerMethodField()
+
+    def get_disabled_use_subtitle_as_transcript(self, obj):
+        """Compute if use_subtitle_as_transcript behavior is disabled.
+
+        Parameters
+        ----------
+        obj : Type[models.Video]
+            The video that we want to serialize
+
+        Returns
+        -------
+        Boolean
+            If there is at least one transcript ready to be shown and no
+            subtitle available the method will return False. Returns True
```

Comment:
The "and no subtitle available" part is not implemented here, right?

Reply:
Yes I choose to only deal with existing transcript, I only need this.